runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -1558,135 +902,151 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadArgMax(const Operator *op, ir
   subg.addOperation(std::move(new_op));
 }
 
-template <typename LoaderDomain, typename SpecificLoader>
-void BaseLoader<LoaderDomain, SpecificLoader>::loadPow(const Operator *op, ir::Graph &subg)
+template <typename LoaderDomain>
+template <typename OpIR, typename... Args>
+const OpIR *BaseLoader<LoaderDomain>::loadOperationTo(const Operator *op, ir::Graph &subg,
```

Comment:
This is a great helper that reduces amount of boilerplate significantly. Why don't submit it (and its uses) first? This would make the reviewing much easier...

Reply:
I divided this PR to two separated commits.